less/index.less

```diff
@@ -54,6 +54,113 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	max-width: 800px;
+	
 }
+p {
+	font-size: 13px;
+	line-height: 16px;
+}
+nav {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	
+	height: 50px;
+	a {
+		margin-right: 15px;
+		color: #0f0f0f;
+		font-size: 12px;
+		
+
+	}
+	a:last-child {
+		margin-right: 45px;
+	}
+}
+Header {
+	height: 440px;
+	background-color: rgb(238, 238, 238);
+	margin: 0 42px 0 42px;
+	border-radius: 4px;
+}
+h1 {
+	font-size: 32px;
+	margin-top: 10px;
+	margin-bottom: 8px;
+	
+}
+
+img {
+	height: 180px;
+}
+
+section {
+	padding-left: 20px;
```

Comment:
these styles on the section padding here are what was breaking the layout out of the area you wanted it